src/Card/Card.tsx

```diff
@@ -190,3 +188,13 @@ const Visual = styled(Box)<{ $visualUrl: string; $visualHeight: string }>`
   ${({ $visualHeight }) =>
     $visualHeight ? `height: ${$visualHeight};` : 'padding-top: 100%;'}
 `
+
+function constructLeadingIcon(leadingIcon: CardProps['leadingIcon']) {
+  if (!leadingIcon) return leadingIcon
```

Comment:
It's to allow for these designs:
<img width="719" alt="Screenshot 2024-10-30 at 14 58 08" src="https://github.com/user-attachments/assets/f13030c1-9dd1-4ab7-9387-37def5a62a8b">

Working more into this, I'm realising there's additional problems with the makeup of the `Card` component for my use case. Instead of making additional changes to make the `Card` component more flexible I'm just going to opt to building the layout from the `children` prop.

Closing out PR, sorry!